### DISCUSSION TITLE
Emgu.CV in Xamarin.Forms (iOS) App linked errors

### DISCUSSION THREAD

**Discussion by milanm** [emgucv/emgucv#697] [Q&A] [answered]
I have commercial license for Emgu.CV and installed from private NuGet to my Xamarin.Forms project (library, Android and iOS app). However, I cannot build iOS app, where I installed Emgu.CV.runtime.ios package. I get **3160 errors**, such as:

`Error Native linking failed, undefined symbol: _cveInputArrayIsMatx. This symbol was referenced by the managed member Emgu.CV.CvInvoke.cveInputArrayIsMatx. Please verify that all the necessary frameworks have been referenced and native libraries linked.`

and for other symbols too.

Any hint why this happens ?

**Comment by milanm** [comment, marked as answer]
When Linked behavior is Link All its working.

**Comment by emgucv** [comment]
Thank you for your update!
